libdnf/dnf-db.c

```diff
@@ -130,14 +132,16 @@ dnf_db_get_dir_for_package(DnfDb *db, DnfPackage *package)
     if (g_strcmp0(instroot, "/") == 0)
         instroot = "";
 
+    version = dnf_package_get_version(package);
```

Comment:
bastards who merged hawkey into libhif without any cleanups. (you remember, that hawkey is not working by refcounting).

Just go ahead and change API into `const gchar *` where you think it should return const. I really think that it should return version, release, location exctly in const.